subsys/net/lib/openthread/platform/radio.c

```diff
@@ -45,6 +47,7 @@ static struct device *radio_dev;
 static struct ieee802154_radio_api *radio_api;
 
 static s8_t tx_power;
+static u16_t channel;
```

Comment:
It's not possible to store your own private data into otInstance? (just asking)

Because here it means OT must work only against 1 15.4 device. Is OT meant to run on N device on one board or?

Reply:
> It's not possible to store your own private data into otInstance? (just asking)

Unfortuantely, there is no API to set private data per instance. `otInstance` is just a pointer to an object of class defined in OpenThread, so unless you modify OpenThread sources there is no option to store anything extra there.

> Because here it means OT must work only against 1 15.4 device. Is OT meant to run on N device on one board or?

I'm afraid that entire OT port is written in a single-instance manner (alarm, flash etc.). OT in general does not support multiple radio interfaces, it doesn't make much sense for me to run multiple OT instances on a single radio either. It perhaps could make sense to run multiple OpenThread instances if each had sepearate 15.4 radio, but is there acutally an use case for that? I don't know, are there any boards with multiple 15.4 interfaces? Anyway, supporting these would require to adjsut entire OT port, not only this part.